scripts/ui/mode_selection.py

```diff
@@ -0,0 +1,170 @@
+import bpy
```

Comment:
Please add `#!/usr/bin/python3` at the start of the file, that helps certain operating systems recognize it as a Python3 file.